src/locutus_util/common.py

```diff
@@ -20,17 +21,22 @@ def get_api_key(api_id):
 
 
 # Dir file paths
-LOGS_PATH = f'data/logs'
-INPUT_PATH = f'data/input'
-ONTOLOGY_DATA_PATH = f'{INPUT_PATH}/ontology_data'
-SIDELOAD_PATH =  f'{INPUT_PATH}/sideload_data'
+HOME_DIR = Path.cwd().resolve()
```

Comment:
I worry about this. That Path.cwd() is the directory where your shell is running, which may not be the same depending on where you are running things. This may be more desirable: 

Path(__file__).parent.parent.resolve()